pipelines/docs/VolumeProvisioning.md

```diff
@@ -0,0 +1,49 @@
+# Volume Provisioning for Kabanero Pipelineruns
+
+Tekton pipelines require a configured volume that is used by the framework to share data across tasks. The build task, which uses Buildah, also requires a volume mount. The minimum required volume size is five Gi.
+
+## Static Persistent Volumes
+
+If you are using an unmanaged cluster, you can set up a persistent volume to be used by the pipelinerun. A simple persistent volume definition is provided in the following example. Update the path and other values in your `pv.yaml` file provided in this repository, to suit your requirements. 
+
+For instance, the following example is configured for local storage. You might edit this file to configure it for NFS, the standard configuration in a production environment.
+
+1. Log in to your cluster. For example, for OKD
+
+```
+oc login <master node IP>:8443
+```
+
+2. Clone the [kabanero-pipeplines repo](https://github.com/kabanero-io/kabanero-pipelines.git)
+
+```
+git clone https://github.com/kabanero-io/kabanero-pipelines
+```
+
+3. Clone the `pv.yaml` file in this repository and apply it.
+
+```
+cd ./pipelines/common
+oc apply -f pv.yaml -n kabanero
+```
+
+## Dynamic Volume Provisioning
```

Comment:
Think we should move this section up before the static one as this is the preferred option if available.

Reply:
I think, point 1 and 2 in above section is generic. I think as a example of static persistent volume we should mention that example  as point 3 in above section after the description of the static persistent volume.